src/post/post.repository.knex.ts

```diff
@@ -679,6 +679,24 @@ export class KnexPostRepository {
         }
     }
 
+    /**
+     * Update the like and/or repost counts of a post
+     *
+     * @param postId ID of the post to update
+     * @param likeCount Updated number of likes (undefined if not updating)
+     * @param repostCount Updated number of reposts (undefined if not updating)
+     */
+    async updateInteractionCounts(
+        postId: number,
+        likeCount?: number,
+        repostCount?: number,
+    ) {
+        await this.db('posts').where({ id: postId }).update({
+            like_count: likeCount,
+            repost_count: repostCount,
+        });
+    }
```

Comment:
Note to reviewer: we're bypassing the Post entity here, as we don't want to remove the `readonly` constraint on `likeCount` / `repostCount`:

https://github.com/TryGhost/ActivityPub/blob/1786516e640907390a3c49ca95b09bb1cdc2a606/src/post/post.entity.ts#L114-L162

Curious if there is a better way?